tests/unit/client_test.py

```diff
@@ -1,15 +1,143 @@
 import os
+
+import sure
```

Comment:
I wouldn't say I hate them, but they're certainly new to me and seem unconventional. Kind of cool actually. Besides readability are there advantages over the [unittest assert methods](https://docs.python.org/2/library/unittest.html#assert-methods)?